aioresponses/core.py

```diff
@@ -247,6 +293,10 @@ def stop(self) -> None:
         self.patcher.stop()
         self.clear()
 
+    @property
+    def calls(self):
```

Comment:
This comment relates [to this comment](https://github.com/pnuckowski/aioresponses/pull/133/files#r330274980)

I do not want to "expose" new property, because it will be tempting (for users) to use them. 
In future the way how aioresponses performs `assert_called_once()` and other `assert_*` methods might change (different structures, etc) and I do now want to "break" compatibility :)